README.md

```diff
@@ -255,3 +255,68 @@ See the [metrics documentation](metrics).
 
 ## Kata Admission controller webhook
 See the [webhook documentation](kata-webhook).
+
+## Using Vagrant to test your code changes
+
+It is strongly recommended that you test your changes locally before opening
+the pull request as this can save people's time and CI resources. Because
+testing Kata Containers involve complex build and setup instructions, scripts
+on the .ci directory are created to easy and provide a reproducible process; but they
```

Comment:
`.ci/`. See https://github.com/kata-containers/kata-containers/blob/main/docs/Documentation-Requirements.md for general advice on quoting in docs.